### PR TITLE
Edit-Copy Display snapping points while moving

### PR DESCRIPTION
**Pull request by vlachoudis**
Display the two cross hairs and show the snapping point while copy

### CODE REVIEW

File: librecad/src/actions/rs_actioneditcopy.cpp [hunk 1]
```diff
@@ -70,6 +70,11 @@ void RS_ActionEditCopy::trigger() {
     RS_DIALOGFACTORY->updateSelectionWidget(container->countSelected(),container->totalSelectedLength());
 }
 
+void RS_ActionEditCopy::mouseMoveEvent(QMouseEvent* e) {
+	if (getStatus()==SetReferencePoint)
+		RS_Vector mouse = snapPoint(e);
+}
+
 void RS_ActionEditCopy::mouseReleaseEvent(QMouseEvent* e) {
```
Comment (dxli) [inline librecad/src/actions/rs_actioneditcopy.cpp]: thanks :+1:

File: librecad/src/actions/rs_actioneditcopy.h [hunk 2]
```diff
@@ -7,7 +7,7 @@
 **
 **
 ** This file may be distributed and/or modified under the terms of the
-** GNU General Public License version 2 as published by the Free Software 
+** GNU General Public License version 2 as published by the Free Software
```
Comment (dxli) [inline librecad/src/actions/rs_actioneditcopy.h]: let's try to avoid empty patches

Comment (vlachoudis) [inline librecad/src/actions/rs_actioneditcopy.h]: Its my configuration on the vim editor. It strips all useless spaces (like at the end of line or before a tab). I have to find out how to revert the file :) Could you simply apply the first part only?

Comment (r-a-v-a-s) [inline librecad/src/actions/rs_actioneditcopy.h]: Note `virtual void mouseMoveEvent(QMouseEvent* e);`

Some git GUIs make it easy to revert the file 
https://git-scm.com/downloads/guis
or https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things

**Comment by dxli** [comment]
okay, I will do it by a patch

**Comment by dxli** [comment]
@vlachoudis 

Added your patch by e3b5395

Please note I also added more C++11 style cleanup: "override" suffix

Looks like we will have to enable C++14 soon, to use std::make_unique(), for example

Thanks!

File: librecad/src/actions/rs_actioneditcopy.h (hunk 2 above)
Comment (vlachoudis) [inline librecad/src/actions/rs_actioneditcopy.h]: Thanks I am new to git.

**Pull request closed without merging**